web/templates/message_edit_notice_tooltip.hbs

```diff
@@ -1,7 +1,13 @@
 <div>
     <div>{{t "View edit history"}}</div>
     {{#if realm_allow_edit_history}}
-        <div class="tooltip-inner-content italic">{{edited_notice_str}}</div>
+        {{#if message_container/msg/local_edit_timestamp}}
```

Comment:
You can also `{{#if message_container/msg/local_edit_timestamp}}` check, since this had already been validated in `edited_notice.hbs`.